api/logging.go

```diff
@@ -0,0 +1,35 @@
+package api
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/runtimeconfig"
+	"github.com/replicatedhq/embedded-cluster/pkg/versions"
+	"github.com/sirupsen/logrus"
+)
+
+func NewLogger() (*logrus.Logger, error) {
```

Comment:
Still need to work on integrating this right? I don't believe it's currently being used.

Reply:
It's actually being used in prod, not applicable.